rails/locale/cy.yml

```
@@ -171,22 +166,29 @@ cy:
       exclusion: wedi cadw
       greater_than: angen bod yn fwy na %{count}
       greater_than_or_equal_to: angen bod yr un maint neu fwy na %{count}
+      in: rhaid bod mewn %{count}
       inclusion: heb fod yn y rhestr
       invalid: heb fod yn nheilwng
       less_than: angen bod yn llai na %{count}
       less_than_or_equal_to: angen bod yr un maint neu lai na %{count}
+      model_invalid: 'Methodd y dilysu: %{errors}'
       not_a_number: heb fod yn rhif
       not_an_integer: heb fod yn rhif llawn
       odd: rhaid bod yn odrif
+      other_than: rhaid bod yn wahanol na %{count}
+      present: rhaid bod yn wag
+      required: rhaid bodoli
       taken: wedi'i gymryd yn barod
       too_long: yn rhy hir (cewch %{count} llythyren ar y fwyaf)
       too_short: yn rhy fyr (rhaid am o leiaf %{count} llythyren)
       wrong_length: gyda maint anghywir o lythrennau (dylai fod yn %{count} llythyren)
     template:
       body: 'Cafwyd broblemau gyda''r meysydd canlynol:'
       header:
+        zero: Atalwyd y %{model} hwn rhag ei gadw gan 0 nam
         one: Atalwyd y %{model} hwn rhag ei gadw gan 1 nam
         two: Atalwyd y %{model} hwn rhag ei gadw gan %{count} nam
+        few: Atalwyd y %{model} hwn rhag ei gadw gan ychydig nam
```

Shouldn’t this still have the `count` variable?

```suggestion
        few: Atalwyd y %{model} hwn rhag ei gadw gan %{count} nam
```